packages/blocks/src/database-block/table/components/selection/selection.ts

```diff
@@ -96,11 +96,12 @@ export class DatabaseSelectionView extends WithDisposable(ShadowlessElement) {
     );
     this._disposables.add({
       dispose: this.eventDispatcher.add('click', context => {
-        const target = context.get('pointerState').event.target;
+        const event = context.get('pointerState').event;
+        const target = event.target;
         if (target instanceof Element && this.isCurrentDatabase(target)) {
           const cell = target.closest('affine-database-cell-container');
           if (cell) {
-            cell?.cell?.enterEditMode();
+            cell?.cell?.enterEditMode(event);
```

Comment:
enterEditMode may be called without an `event` in future, Can we not depend an `event` here

Reply:
Here I want to trigger the click of the link when clicking the external area of the link, so I need to get the relevant elements.

<img width="327" alt="image" src="https://github.com/toeverything/blocksuite/assets/15389209/107defbb-563c-4a02-a4b7-9d63a08d3ab8">

---

Related code:

https://github.com/toeverything/blocksuite/pull/3187/files#diff-7f1b6d33e4075b6a3dc56f7a12e6d0bfca7122157175c37bfe3eaad36c347ec7R127-R135

Reply:
Get!